Xamarin.Forms.Platform.Android/FormsApplicationActivity.cs

```diff
@@ -83,28 +79,17 @@ public void SetPage(Page page)
 
 		protected void LoadApplication(Application application)
 		{
-			if (application == null)
-				throw new ArgumentNullException("application");
+			if (_application != null)
+				_application.PropertyChanged -= AppOnPropertyChanged;
 
-			(application as IApplicationController)?.SetAppIndexingProvider(new AndroidAppIndexProvider(this));
-
-			_application = application;
+			_application = application ?? throw new ArgumentNullException(nameof(application));
+			((IApplicationController)application).SetAppIndexingProvider(new AndroidAppIndexProvider(this));
```

Comment:
Same as above. why the change? and maybe use the method directly.

Reply:
This change was only to be uniform between the two platforms.